templates/openwisp2/urls.py

```diff
@@ -19,6 +19,11 @@
     {% if openwisp2_firmware_upgrader %}
     url(r'^', include('openwisp_firmware_upgrader.urls')),
     {% endif %}
+    {% if openwisp2_radius %}
+    url(r'^', include('openwisp_radius.urls')),
+    url(r'^api/v1/', include('openwisp_users.api.urls')),
+    url(r'^api/v1/', include('openwisp_utils.api.urls')),
+    {% endif %}
```

Comment:
I did not take out the api out of the if condition because `rest_framework.authtoken` needs to be added to openwisp-utils to be used with openwisp-radius, we can do it in https://github.com/openwisp/ansible-openwisp2/issues/234 separately, what do you think?

Reply:
hopefully now that https://github.com/openwisp/ansible-openwisp2/commit/694e7198bfa3dbb8ca42d168a46c3a84af4fe21d is merged we shouldn't need this, right?